src/fft128/mod.rs

```diff
@@ -2112,7 +2306,7 @@ mod tests {
     #[test]
     fn test_product_avx512() {
         if let Some(simd) = V4::try_new() {
```

Comment:
don't we need additional tests for V4x2 ?

Reply:
both `V4` and `V4x2` are used in the current implementation

Reply:
ok perfect !